database/index.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-const mongoUri = 'mongodb://localhost/practice';
+const mongoUri = 'mongodb://localhost:27017/practice';
```

Comment:
The mongodb:// should default to this port. If you're using a non-standard port maybe add a 
`const SERVER_PORT = 'something'`
And reference it here. This is handy for little things like this need changing. If it's not going to need changed down the road, though, please disregard this comment.